### ISSUE TITLE
Risk functional vs Loss function

### ISSUE THREAD

**Issue by sgomezr** [Statistical-Connectomics-Sp15/intro#99]
What is exactly the difference between a risk functional and a loss function? Last class, we discussed MSE (minimum squared error) and ARI (Arbitrary rand index) as  good examples for the loss function, and defined the risk functional as l:ARI. Does this means that the risk functional is dependent on the loss function?

**Comment by kristinmg** [comment]
This is confusing me as well so I'd appreciate if someone could explain

**Comment by itsthefedora** [comment]
(Potential duplicate of https://github.com/Statistical-Connectomics-Sp15/intro/issues/97)

**Comment by sgomezr** [comment]
Not really, because I want more of a comparison. Especially because I have noticed that some functions (like the MSE) can be sometimes used as decision rule (if using Bayes), loss function and risk functional...

**Comment by itsthefedora** [comment]
I think there's some wiggle room in the particulars of the formalism, but generally, *loss* is measuring how bad your protocol is at assigning actions to a *particular* sample, where as *risk* is measuring how bad your protocol is at assigning actions *overall* to samples that are *realizations* of a specific random variable in your model class. So, your decision protocol might be really good at figuring out the "truth" for a particular graph g---i.e., Loss(g) ~ 0---but terrible at figuring out the "truth" for graphs drawn from a particular distribution---e.g., Risk( ErdősRényi(2, 0.01) ) >> 0.

**Comment by akim1** [comment]
This is my understanding of loss and risk. if you were to pull on a slot machine once, that would give you a loss (e.g., the amount of money you won/lost). If you were to pull on a slot machine for a sufficiently large number of time, you will get risk which tells you, on average, how much money you will lose/win.

The easiest way to think of this is that whatever decision framework you will have will sometimes give you wrong answers and sometimes give you right answers (i.e., loss). On average, you want to get the right answer (i.e., risk) or it wouldn't be a very useful model.

**Comment by adjordan** [comment]
I believe that is the correct answer. Your risk is the probability of a loss.

**Comment by sgomezr** [comment]
Thanks guys! ^^

**Comment by jovo** [comment]
yup, more generally, risk can be defined as any functional of the
distribution of loss, with respect to the distribution of the data.

so, \int f(loss(x,y)) * prob(x) dx,
where y is the truth, and f(.) is some function, such as expectation or
median.

On Fri, Feb 20, 2015 at 2:05 PM, Sandra Gomez <notifications@github.com>
wrote:

>  Thanks guys! ^^
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Statistical-Connectomics-Sp15/intro/issues/99#issuecomment-75299434>
> .
>



-- 
the glass is all full:  half water, half air.
openconnecto.me, we're hiring!
<https://docs.google.com/document/d/14SApYAzxF0Ddqg2ZCEwjmz3ht2TDhDmxyZI2ZP82_0U/edit?usp=sharing>
, jovo.me, my calendar
<https://www.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York>
